cmd/kube-controller-manager/app/options/options_test.go

```diff
@@ -461,8 +453,6 @@ func TestApplyTo(t *testing.T) {
 	expected := &kubecontrollerconfig.Config{
 		ComponentConfig: kubectrlmgrconfig.KubeControllerManagerConfiguration{
 			Generic: cmconfig.GenericControllerManagerConfiguration{
-				Port:            10252,     // Note: InsecureServingOptions.ApplyTo will write the flag value back into the component config
```

Comment:
same question for these... where is the listen address/port set if not here?

Reply:
answered above https://github.com/kubernetes/kubernetes/pull/96216#discussion_r522169557